ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-settings/user-settings.tsx

```diff
@@ -107,6 +107,25 @@ class UserSettings extends React.Component<Props, State> {
   render() {
     const { extensions } = this.props
     const { component } = this.state
+    const extensionComponents = extensions.map((extension: SettingsComponent) => {
+      if(extension.custom) {
+        return extension.children
+      } else {
+        return (
+          <NavigationButton
```

Comment:
✏️ styling here is relatively simple, but you could export `NavigationButton` for use downstream. That way buttons will stay consistent if there are any styling changes